### DISCUSSION TITLE
Having a spot of bother getting Stellarium up and running right on Ras Pi 5.

### DISCUSSION THREAD

**Discussion by BoydieUK** [Stellarium/stellarium#3639] [Q&A]
Hi,
I'm having a spot of bother getting Stellarium up and running on a Pi5. 

Using standard snap and the --edge argument it installs fine and loads up, but without any menus and the keyboard shortcut for the config doesn't work.
Most other keyboard shortcuts do work (eg, Ground ,time shift, F11 for full or window screen).

Without the --edge it just stalls on load, guessing the gl drivers/shaders are not right on the pi yet.

I'm just building up a little raspi telescope control system as a little project and wanted to run Stellarium rather than kstars. 

Can anyone give me some pointers on the above.

Thanks
Ste

**Comment by gzotti** [comment]
Hi! I have no RPi5 yet to test and no experience with snap problems (and we don't provide snaps AFAIK). On RPi4 you just follow compilation instructions for Linux. I expect the same goes for RPi5.

**Comment by alex-w** [comment]
Please try switching to use X11 session

**Comment by BoydieUK** [comment]
Thanks for your replies. 
Strange that you say theres no snap, yet its listed on the main site page
<img width="469" alt="image" src="https://github.com/Stellarium/stellarium/assets/161154511/10730b25-cc20-4185-b781-a58fe0726e33"> 

Not sure x11 will work, damn pi started to warm up a tad too much for my liking when I had all the parts open for the control I wanted. 
I'm going to revert back to using m$ windoze system which has a little more power behind it. Also means the mrs (whose telescope it is) will be able to run it.   
 
 Thanks again for your replies

**Reply by gzotti** [reply]
OK, we do link snaps. I don't know who builds them, though.
The Pi5 seems to really require a fan, at least in most scenarios where it is used when a Pi4 has not enough power. You may want to reduce Stellarium's idle framerate to keep it cooler.
